pkg/sql/testdata/pg_catalog_tables_from_postgres.json

```diff
@@ -3657,6 +3664,178 @@
+      "datname": {
+        "oid": 19,
+        "dataType": "name",
+        "expectedOid": null,
+        "expectedDataType": null
+      },
+      "deadlocks": {
+        "oid": 20,
+        "dataType": "int8",
+        "expectedOid": null,
+        "expectedDataType": null
+      },
+      "numbackends": {
+        "oid": 23,
+        "dataType": "int4",
+        "expectedOid": null,
+        "expectedDataType": null
+      },
+      "stats_reset": {
+        "oid": 1184,
+        "dataType": "timestamptz",
+        "expectedOid": null,
+        "expectedDataType": null
+      },
+      "temp_bytes": {
+        "oid": 20,
+        "dataType": "int8",
+        "expectedOid": null,
+        "expectedDataType": null
+      },
+      "temp_files": {
+        "oid": 20,
+        "dataType": "int8",
+        "expectedOid": null,
+        "expectedDataType": null
+      },
+      "tup_deleted": {
+        "oid": 20,
+        "dataType": "int8",
+        "expectedOid": null,
+        "expectedDataType": null
+      },
+      "tup_fetched": {
+        "oid": 20,
+        "dataType": "int8",
+        "expectedOid": null,
+        "expectedDataType": null
+      },
+      "tup_inserted": {
+        "oid": 20,
+        "dataType": "int8",
+        "expectedOid": null,
+        "expectedDataType": null
+      },
+      "tup_returned": {
+        "oid": 20,
+        "dataType": "int8",
+        "expectedOid": null,
+        "expectedDataType": null
+      },
+      "tup_updated": {
+        "oid": 20,
+        "dataType": "int8",
+        "expectedOid": null,
+        "expectedDataType": null
+      },
+      "xact_commit": {
+        "oid": 20,
+        "dataType": "int8",
+        "expectedOid": null,
+        "expectedDataType": null
+      },
+      "xact_rollback": {
+        "oid": 20,
+        "dataType": "int8",
+        "expectedOid": null,
+        "expectedDataType": null
+      }
+    },
+    "pg_stat_database_conflicts": {
+      "confl_bufferpin": {
+        "oid": 20,
+        "dataType": "int8",
+        "expectedOid": null,
+        "expectedDataType": null
+      },
+      "confl_deadlock": {
+        "oid": 20,
+        "dataType": "int8",
+        "expectedOid": null,
+        "expectedDataType": null
+      },
+      "confl_lock": {
+        "oid": 20,
+        "dataType": "int8",
+        "expectedOid": null,
+        "expectedDataType": null
+      },
+      "confl_snapshot": {
+        "oid": 20,
+        "dataType": "int8",
+        "expectedOid": null,
+        "expectedDataType": null
+      },
+      "confl_tablespace": {
+        "oid": 20,
+        "dataType": "int8",
+        "expectedOid": null,
+        "expectedDataType": null
+      },
+      "datid": {
+        "oid": 26,
+        "dataType": "oid",
+        "expectedOid": null,
+        "expectedDataType": null
+      },
+      "datname": {
+        "oid": 19,
+        "dataType": "name",
+        "expectedOid": null,
+        "expectedDataType": null
+      }
+    },
```

Comment:
i would have expected all the other pg_stat tables to start appearing here. do you know why they did not?

Reply:
Using another way to filter this columns here #66560